gnowledge/gstudio
# Quiz module

## Implementation of Quiz Module: ##

Prerequisites: Run filldb, STs_run1.csv, ATs.csv, RTs.csv, STs_run2.csv scripts

*What to test:*

1. Go to Quiz GAPP
2. Create QuizItem (i.e Question)
3. Go to Course/CourseEvent authoring page and add the newly created QuizItem.
4. Check in course player, click on quizitem from LHS. This will load the quizplayer based on quiz-type.

* Based on Check-answer flag in quiz-item, 'Check' button will be displayed in course player
* Based on Show-answer flag in quiz-item, 'Show Answer' button will be displayed in course player
* On 'Submit', a new instance of QuizItemPost will be created. This instance will hold the user's given answer as new GAtrribute: 'quizitempost_user_given_ans'
